src/panels/lovelace/cards/hui-map-card.js

```diff
@@ -11,7 +11,23 @@ import computeStateName from "../../../common/entity/compute_state_name";
 import debounce from "../../../common/util/debounce";
 import parseAspectRatio from "../../../common/util/parse-aspect-ratio";
 
+export const Config = {
```

Comment:
I am so confused. You are using this as an interface but you also have it contain defaults but those are not used??